website/pages/docs/from-1.5/functions/index.mdx

```diff
@@ -9,6 +9,10 @@ description: |-
 
 # Built-in Functions
 
+-> **Note:** This page is about HCL2 in Packer 1.5 and later. HCL2 support for
+Packer is still in Beta. For the old-style stable configuration language see
+[template docs](http://localhost:3000/docs/templates).
```

Comment:
localhost link.

Reply:
hey it works on my machine, 🤦. Good call 🙂